File: javaToC++/ex7.cpp
/*
Class: CPSC 122-01
Team Member 1: Paul De Palma
Team Member 2: None
Submitted By: Paul De Palma
GU Username: depalma
File Name: ex7.cpp
Program is a first look at command line arguments 
To Build: g++ ex7.cpp -o ex7
Sample Execution: ./ex7 1 2 3 4 5 
*/

#include <iostream>
using namespace std;

//argc holds the number of command line arguments
//argv is an array of the arguments

int main(int argc, char* argv[])
{
 cout << endl;

 cout << "Total Number Command Line Arguments: " << argc << endl << endl;
 for (int i = 0; i < argc; i++)
   cout << "Argument " << i << ": " << argv[i] << endl;

 return 0;

}  

 

File: JavaToC++/ex7.java
/*
Class: CPSC 122
Section: 01
Team Member 1: Paul De Palma
Team Member 2: None
Submitted By: Paul De Palma
GU Username: depalma
File Name: ex7.java
Program illustrates command line arguments
arrays, random numbers, global constants
To Build: javac ex7.java
To Execute: java ex7 
*/

import java.util.Scanner;
import java.util.Random;
import java.lang.Integer;

public class ex7 
{
  public static void main(String[] args)
  {
    System.out.println("Total number of command line arguments: " + args.length); 
    for (int i = 0; i < args.length; i++)
      System.out.println("Argument " + i + ": " + args[i]);
  }

}
